src/Components/dashboard/Dashboard.js

```diff
@@ -0,0 +1,156 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { Avatar, Button } from "@mui/material";
+import ViewModuleIcon from "@mui/icons-material/ViewModule";
+import ViewStreamIcon from "@mui/icons-material/ViewStream";
+import PersonIcon from "@mui/icons-material/Person";
+import AddCircleIcon from "@mui/icons-material/AddCircle";
+
+import eventsInformation from "../../utils";
```

Comment:
rename it from `eventsInformation` to  `mockedEvents` in here we should only import the events object, so go the utils file and export it, this is because maybe in the future we want to export multiple things from the utils file